### DISCUSSION TITLE
how to page jump to top stop?

### DISCUSSION THREAD

**Discussion by nurullah7733** [AdeleD/react-paginate#510] [Q&A]
When i click a page number, prev or next button then page jump to top. how to disable this?

**Comment by sneha9231** [comment]
Make use of the **preventDefault()** method of the event object in your click event handler. HandleClick method prevents the default behavior of the anchor tag or button by calling e.preventDefault() , like this : `handleClick = (e) => {
    e.preventDefault();`
